src/components/date.js

```diff
@@ -4,11 +4,11 @@
  * See the accompanying LICENSE file for terms.
  */
 
-import React, {Component, PropTypes} from 'react';
+import * as React from 'react';
```

Comment:
I had to make this switch because Rollup doesn't allow the `intro` to go _before_ the `globalVar['default']` check.